[PATCH] Fix STRICT_ANSI vs POSIX

* build-aux/ltmain.in (func_mode_link): Also check for _POSIX
as well as __STRICT_ANSI__ to avoid re-definitions.
---
 build-aux/ltmain.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build-aux/ltmain.in b/build-aux/ltmain.in
index 3e2adfb1..5f9af8c2 100644
--- a/build-aux/ltmain.in
+++ b/build-aux/ltmain.in
@@ -3670,7 +3670,7 @@ EOF
 
 /* declarations of non-ANSI functions */
 #if defined __MINGW32__
-# ifdef __STRICT_ANSI__
+# if defined(__STRICT_ANSI__) && !defined(__MINGW64_VERSION_MAJOR) || defined(_POSIX_)
 _CRTIMP int __cdecl _putenv (const char *);
 # endif
 #elif defined __CYGWIN__
